src/main/java/io/debezium/connector/cassandra/CassandraConnectorTask.java

```
@@ -70,6 +70,11 @@ public CassandraConnectorTask(CassandraConnectorConfig config) {
 
     void run() throws Exception {
         try {
+            if (!config.validateAndRecord(config.ALL_FIELDS, LOGGER::error)) {
+                LOGGER.error("Failed to start connector with invalid configuration (see logs for actual errors)");
+                return;
```

This should throw a `CassandraConnectorConfigException` instead of just return.